Project.toml

```
@@ -11,7 +11,7 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 Distributions = "0.23, 0.24, 0.25"
-GLM = "^1.3.11"
+GLM = "^1.4.0"
```

```suggestion
GLM = "1.4"
```
This means GLM version ≥ 1.4. Specifically, `[1.4.0, 1.5.0)` see https://pkgdocs.julialang.org/v1/compatibility/.

---

Yeah, good catch.